Atomic/install.py

```diff
@@ -80,8 +86,31 @@ def cli(subparser):
 
 
 class Install(Atomic):
-    def __init__(self):
-        super(Install, self).__init__()
+    def build_rpm_for_docker_backend(self, image_id, temp_dir, labels):
```

Comment:
this looks like Docker specific stuff, we should move it to `backends/_docker.py`